www/js/services.js

```diff
@@ -48,12 +48,10 @@ angular.module('pvta.services', ['ngResource'])
 
 
 .factory('StopList', function(){
-  var stopsList = [];
-  var pushToList = function(stop){
-    stopsList.push(stop);
-  };
+  var stopsList = []; 
+
   var pushEntireList = function(list){
-    stopsList = stopsList.concat(list);
+    stopsList = stopsList.concat(_.uniq(list, true, 'Name'));
```

Comment:
@dfaulken I took out the line that sorts stops by name, because we want to sort them by distance from user instead. However, this line still works, and I believe that's because the stops are ordered by distance, meaning all three Haigis Mall items are next to each other in the array. This is probably all Underscore is looking for when asking if the array is sorted.